addons/block_code/ui/blocks/utilities/template_editor/template_editor.gd

```diff
@@ -87,26 +88,36 @@ func _update_from_format_string():
 		child.queue_free()
 
 	var match_id = 0
+	var is_collapsable = false
+	var collapsable := CollapsableSettingsScene.instantiate() as CollapsableSettings
 	for item in BlockDefinition.parse_display_template(format_string):
+		var hidden = item.has("hidden") and item.get("hidden")
+		if hidden:
+			is_collapsable = true
 		if item.has("label"):
-			_append_label(item.get("label"))
+			_append_label(collapsable if hidden else _container, item.get("label"))
 		elif item.has("in_parameter"):
-			_append_input_parameter(item.get("in_parameter"), match_id)
+			_append_input_parameter(collapsable if hidden else _container, item.get("in_parameter"), match_id)
 		elif item.has("out_parameter"):
-			_append_output_parameter(item.get("out_parameter"), match_id)
+			_append_output_parameter(collapsable if hidden else _container, item.get("out_parameter"), match_id)
 		elif item.has("const_parameter"):
-			_append_const_parameter(item.get("const_parameter"), match_id)
+			_append_const_parameter(collapsable if hidden else _container, item.get("const_parameter"), match_id)
```

Comment:
The `collapsable if hidden else _container` is repeated 4 times. Is better to move it to a variable before the if/elif/elif/elif block.